ds-collector/ds-collector

```diff
@@ -427,7 +430,7 @@ s3_push_artifact() {
 
 create_complete_marker() {
   artifact_count="${#artifacts[@]}"
-  node_count="${#cassandraNodes[@]}"
+  node_count="$(echo $cassandraNodes | wc -w)"
```

Comment:
adding the tests smoke out this failure (`$cassandraNodes` is not an array)

so not strictly part of this PR, but included anyway